src/middlewares/winstonLogger.ts

```diff
@@ -0,0 +1,48 @@
+import winston from 'winston';
```

Comment:
Logger isn't a middleware, so it shouldn't be put into middlewares directory.

Reply:
fixed